joechz/FreeRTOS
# Created STM32_CORTEX_M4_GENERIC from CORTEX_LM3S6965_GCC_QEMU

Also changed the machine time from Cortex M3 to Cortex M4


Added support for STM32_CORTEX_M4_GENERIC machine

Description
-----------
As discussed we should have STM32_CORTEX_M4_GENERIC machine (i.e., folder) created with contents copied from CORTEX_LM3S6965_GCC_QEMU. This was missing in your last release.

Test Steps
-----------
Navigate to Demo folder and manually inspect if you have STM32_CORTEX_M4_GENERIC folder in it. Also try to import it using Eclipse and build it.

Related Issue
-----------
Refer Description above.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
